src/applications/hca/actions.js

```diff
@@ -98,10 +118,55 @@ export function getEnrollmentStatus(formData) {
       ) {
         callFake404(dispatch);
       } else {
-        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.canceledDeclined);
+        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.enrolled);
       }
     } else {
       callAPI(dispatch, formData);
     }
   };
 }
+
+export function getDismissedHCANotification() {
+  return dispatch => {
+    dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_STARTED });
+    const url = `/notifications/dismissed_statuses/form_10_10ez`;
+
+    apiRequest(
+      url,
+      null,
+      data =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED, data }),
+      ({ errors }) =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_FAILED, errors }),
+    );
+  };
+}
+
+export function setDismissedHCANotification(status, statusEffectiveAt, update) {
+  return dispatch => {
+    dispatch({
+      type: SET_DISMISSED_HCA_NOTIFICATION_STARTED,
+      data: statusEffectiveAt,
+    });
+    if (update) {
+      apiRequest('/notifications/dismissed_statuses/form_10_10ez', {
```

Comment:
We aren't going to do anything if these calls succeed or fail. We are just acting as though they will succeed. There is no reason to show the user error messages if something goes wrong with the API and we aren't able to remember that the user has dismissed a particular notification. The only drawback of an API failure on these calls is that it's possible that a user will see a notification they already dismissed.

Reply:
we could fire an error action and/or report it to sentry 

also i don't see where this function is being used.

Reply:
it's being used in my PR that will follow this one. I broke my branch into two commits to make the PRs easier to review